lib/pdf/core/page.rb

```diff
@@ -167,6 +173,9 @@ def init_new_page(options)
         @dictionary = document.ref(:Type        => :Page,
                                    :Parent      => document.state.store.pages,
                                    :MediaBox    => dimensions,
+                                   :CropBox     => dimensions,
+                                   :BleedBox    => dimensions,
```

Comment:
I would suggest relying more on official documents. Adobe has a good PDF Reference document. I use v 1.7. It covers all the previous versions to as PDF is backwards compatible format.

Here's an illustration from the chapter "Page Boundaries":
![screen shot 2015-05-26 at 1 45 37 pm](https://cloud.githubusercontent.com/assets/7405/7810813/044f0604-03ae-11e5-90ad-9d43b3de3818.png)